render/query.go

```diff
@@ -249,8 +262,9 @@ func (r *Reply) getDataAggregated(ctx context.Context, cfg *config.Config, tf Ti
 		}
 
 		if targets.isReverse {
```

Comment:
No need `if` here, just `m = metricList[n]` when suggestions are accepted

Reply:
After this step, m must be unreversed for use in steps map. See code after 
https://github.com/lomik/graphite-clickhouse/blob/master/render/query.go#L375